src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -310,10 +321,12 @@ std::string tenantFromDb(std::string& database)
 {
   std::string r;
   std::string prefix  = dbPrefix + "-";
+
   if (strncmp(prefix.c_str(), database.c_str(), strlen(prefix.c_str())) == 0)
   {
     char tenant[MAX_SERVICE_NAME_LEN];
```

Comment:
I think at the end we have decided to change MAX_SERVICE_NAME_LEN to 50, use MAX_SERVICE_NAME_LEN + 1 to size the array and find a better fix for this in the context of #1403.